### DISCUSSION TITLE
Plugin Store: UI Enhancement - Plugin Landing Page

### DISCUSSION THREAD

**Discussion by richhayler** [craftcms/cms#8653] [Ideas] [answered]
### Description
There should be a link at the top of the plugin store navigation to take you back to the plugin store landing page rather than having to click the “plugin store” link in the main nav within the CP.

_I notice that you can click on the Plugin Store section title but this is not consistent behaviour with other section titles used throughout the control panel, e.g. if I navigate to “users”, select a user group and then try to click the “users” section title which in this case is not a link that takes me back to the “users” landing page. Furthermore the “Plugin Store” section title link does not appear as a clickable link._

**Comment by brandonkelly** [comment, marked as answer]
As of Craft 4 the landing page now has a dedicated “Discover” nav item.
